test/cli/test-other.py

```diff
@@ -888,13 +888,13 @@ def test_premium_with_relative_path(tmpdir):
                 """.replace('NAME', product_name))
 
     # should be fine now
-    exitcode, stdout, stderr = cppcheck([cppcheck_exe, '--premium=misra-c++-2008', test_file])
+    exitcode, stdout, stderr = cppcheck(['--premium=misra-c++-2008', test_file], None, True, cppcheck_exe)
```

Comment:
rather than providing all arguments explicitly it's better to write:

    exitcode, stdout, stderr = cppcheck(['--premium=misra-c++-2008', test_file], cppcheck_exe=cppcheck_exe)